test/suites/nodeclaim/nodeclaim_test.go

```diff
@@ -54,8 +54,8 @@ var _ = Describe("StandaloneNodeClaim", func() {
 			},
 		})
 		env.ExpectCreated(nodeClass, nodeClaim)
+		nodeClaim = env.EventuallyExpectRegisteredNodeClaimCount("==", 1)[0]
```

Comment:
this scenario turned flaky after the v1.4.0 release, i suspect its because of LRO things, and this doesnt really fix it, unlike other tests, this assumes we will create the nodeclaim with that name, but in the case of Zonal Allocation errors we sometimes(1/15 runs) see this one fail. 


I suspect that we need to rewrite these tests to support the new paradigm

Reply:
cc: @matthchr